### PR TITLE
DRUP-997: #ready new equipment block on hp using views

### PR DESCRIPTION
**Pull request by emb03**
@z3cka This the first attempt to get this block 'content' into code using views. Not sure you will agree with my approach, there may be an easier way to do this. plz review, thx.

### CODE REVIEW

**Comment by z3cka** [comment]
cool, will check it out after the responsive commit prep

**Comment by emb03** [comment]
@z3cka WST is going to ask about this tomorrow. Will you have any time to review before then?

**Comment by z3cka** [comment]
ya, will review/rebase and deploy to `www-test` today

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

File: www/sites/all/modules/features/uclalib_homepage/uclalib_homepage.info [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = UCLALIB Homepage
 description = Homepage panel.
-core = 7.x-1.0-beta1
+core = 7.x-1.0-beta2
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_homepage/uclalib_homepage.info]: @emb03 this should be 
```
core = 7.x
project = uclalib_homepage
version = 7.x-1.0-beta2
```
sorry I missed this.

**Pull request merged**